apisix/cli/ngx_tpl.lua

```diff
@@ -582,9 +582,6 @@ http {
         {% if proxy_protocol and proxy_protocol.listen_http_port then %}
         listen {* proxy_protocol.listen_http_port *} default_server proxy_protocol;
         {% end %}
-        {% if proxy_protocol and proxy_protocol.listen_https_port then %}
```

Comment:
We can't simply delete the listen_https_port feature. Maybe we should provide an implementation without http2?

Reply:
my mistake, just remove ssl.enable_http2